position_controllers/test/test_load_joint_group_position_controller.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright 2020 PAL Robotics SL.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+#include <memory>
+
+#include "controller_manager/controller_manager.hpp"
+#include "test_robot_hardware/test_robot_hardware.hpp"
+
```

Comment:
From a pure unit testing perspective, having a test on forwarding commands may not be sensible as this would be duplicating forward command group controller tests *but* I think there's value in it from an integration test perspective. Would it be too much trouble to add one following the load test? We could slowly turn these into some higher level integration tests.

Reply:
I'm not sure I follow.

You'd like to have tests on the actual forwarded command for the `JointGroupPositionController`? Just like in [test_forward_command_controller.cpp](https://github.com/ros-controls/ros2_controllers/blob/master/forward_command_controller/test/test_forward_command_controller.cpp)?

Reply:
Yes

Reply:
https://github.com/ros-controls/ros2_controllers/issues/100